Docs/Practitioners/Profile.md

```diff
@@ -0,0 +1,215 @@
+| schoolType | [CodedDescription](CodedDescription.md) | The degree information |
+| city | string | The name of the city where the school is located |
+| stateOrProvince | [Region](#region) |  The state or province where the school is located | 
+
+## Identification
+
+Holds the federation id of the medical professional
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| ssnLast4  | string | The last four digits of the practitioner's SSN |
+| npi | string | The Canadian unique identifier |
+| usmleId | string | The practitioner's Usmle identification number |
+| birthDate | DateTime | The practitioner's date of birth |
+| birthCity | string | The practitioner's birth city |
+| birthStateOrProvince | [Region](#region) | The practitioner's birth state or province |
+| gender | string | The practitioner's gender |
+
+## MedicalEducation
+
+Holds information surrounding a person's medical education
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| school  | [School](#school) | School information |
+| beginDate | DateTime | The date a person started attending  a school |
+| endDate | DateTime | The date a person finished attending a school |
+| degree | [CodedDescription](CodedDescription.md) | An object containing degree information |
+| graduationDate | DateTime | The date the person who attended the school graduated | 
+
+## Name
+
+Holds name information
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| isLegal  | boolean | Indicates whether the provided name is their legal name |
+| firstName| string | The person's first name |
+| middleName | string | The person's middle name |
+| lastName | string | The person's last name |
+| suffix | string | The suffix the person uses |
+
+## OtherTraining
+
+Holds the other training information of the medical professional
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| program | [Program](#program) | Program information |
+| specialty | [Specialty](#specialty) | Specialty information |
+| programType | string | The program type |
+| trainingStatus | string | The practitioner's training status |
+| beginDate | DateTime | The date the practitioner started the training |
+| endDate | DateTime | The date the practitioner finished the training |
+
+## Phone
+
+Holds a person's phone number
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| isPrimary  | boolean | Indicates whether the provided email address is the person's primary |
+| phoneType | string | Indicates the phone type. Ex: Business, Home, Mobile |
+| phoneNumber | string | The person's phone number |
+| extension | string | The extension for the phone number, if necessary |
+
+## Program
+
+Holds a program's relevant data 
+
+| Name | Type   | Description |
+| ---- | ------ | ----------- |
+| hospitalName | string | The hospital's name |
+| affiliatedInstitution | string | The institution the program is affiliated with, if any |
```

Comment:
The affiliated institution is always set IIRC. Hospital name may not be though. I'd have to look at the data. Charla generally knows these rules.